server/akka-http-server/src/main/scala/sttp/tapir/server/akkahttp/AkkaToResponseBody.scala

```diff
@@ -55,10 +55,16 @@ private[akkahttp] class AkkaToResponseBody(implicit m: Materializer, ec: Executi
       case RawBodyType.ByteArrayBody   => HttpEntity(ct, r)
       case RawBodyType.ByteBufferBody  => HttpEntity(ct, ByteString(r))
       case RawBodyType.InputStreamBody => streamToEntity(ct, contentLength, StreamConverters.fromInputStream(() => r))
+      case RawBodyType.InputStreamRangeBody =>
+        val resource = r
+        val initialStream = StreamConverters.fromInputStream(resource.inputStreamFromRangeStart, ChunkSize)
```

Comment:
why are we using `ChunkSize` here, but not in the `InputStreamBody` case?

Reply:
Yes, thanks for spotting this.